gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -308,6 +308,8 @@
    */
   public static final String MR_JOB_ROOT_DIR_KEY = "mr.job.root.dir";
   public static final String MR_JOB_MAX_MAPPERS_KEY = "mr.job.max.mappers";
+  public static final String MR_INCLUDE_TASK_COUNTERS_KEY = "mr.include.task.counters";
+  public static final boolean DEFAULT_MR_INCLUDE_TASK_COUNTERS = Boolean.TRUE;
```

Comment:
Why not set it off by default, or just don't report task metrics through Hadoop counters at all. Anytime a user uses Task metrics and runs over 120 map tasks, their job will fail, which could happen pretty frequently.